scalameta/semanticdb-scalac/src/main/scala-2.12.4/scala/meta/internal/SemanticdbAnalyzer.scala

```diff
@@ -0,0 +1,1151 @@
+// NOTE: has to be this package or otherwise we won't be able to access private[typechecker] methods
```

Comment:
Is there a diff of what changed in comparison with https://github.com/scalameta/scalameta/pull/1144/commits/b62db4f2e9ca0a468dd21f8589959f19156f8d06? I'm a bit confused that we have two commits that add the same file. Do they come from different branches?

Reply:
I rebased the commits so the history is cleaner now https://github.com/scalameta/scalameta/commit/7acf345c3d00706dcabdd947a67476f4a7bbffe4 I recommend you keep the full 2.12.3..2.12.4 diff alongside to double-check it's correct.

Reply:
Very nice! Thank you, that's exactly what I wanted. I'll review the diff today.